bot/exts/fun/off_topic.py

```diff
@@ -0,0 +1,202 @@
+        self.ot_names: Dict[str:int] = ...
+
+        self.bot.loop.create_task(self._cache())
+        self.bot.loop.create_task(self.update_ot_channel_name())
+
+    async def _cache(self) -> None:
+        """Get all off topic channel names."""
+        await self.bot.wait_until_ready()
+
+        self.ot_channel: TextChannel = self.bot.get_channel(
+            constants.Channels.off_topic
+        )
+        self.ot_names = dict(
+            await db_fetch(self.bot.db_pool, "SELECT * FROM offtopicnames")
+        )
+
+    @group(name="offtopicnames", aliases=("otn",), invoke_without_command=True)
+    async def off_topic_names(self, ctx: Context) -> None:
+        """Commands for managing off-topic-channel names."""
+        await ctx.send_help(ctx.command)
+
+    @off_topic_names.command(name="add", aliases=("a",))
+    async def add_ot_name(self, ctx: Context, *, name: OT_Converter) -> None:
+        """Add off topic channel name."""
+        if name in self.ot_names:
+            await ctx.send(f":x:`{name}` already exists!")
+            return
+
+        similar_names = self._find(name)
+
+        if similar_names:
+            embed = self._find_embed_builder(
+                similar_names[:5], "Found similar existing names :name_badge:"
+            )
+            embed.set_footer(text="Do you still prefer to add the off topic name?")
+            confirmation_msg = await ctx.send(embed=embed)
+
+            await confirmation_msg.add_reaction(CHECK_MARK_EMOJI)
+            await confirmation_msg.add_reaction(CROSS_MARK_EMOJI)
+
+            def check(reaction: Reaction, user: User) -> bool:
+                return reaction.message == confirmation_msg and user == ctx.author
+
+            async def _exit() -> None:
+                await ctx.send(f"Off topic name `{name}` not added.")
+
+            try:
+                reaction, _ = await self.bot.wait_for(
+                    "reaction_add", check=check, timeout=60.0
+                )
+            except asyncio.TimeoutError:
+                return await _exit()
+
+            if str(reaction.emoji) == CROSS_MARK_EMOJI:
+                return await _exit()
+
+        # ot_ins = await OT_Model.create(name=name)
+        await db_execute(
+            self.bot.db_pool, "INSERT INTO offtopicnames VALUES ($1)", name
+        )
+        self.ot_names[name] = 0
+
+        await ctx.send(f":ok_hand: `{name}` has been added!")
+
+    @off_topic_names.command(name="delete", aliases=("d", "r", "remove"))
+    async def delete_ot_name(self, ctx: Context, *, name: OT_Converter) -> None:
+        """Delete off topic channel name."""
+        # otn_instance = (
+        #     await OT_Model.filter(name=name) if OT_Model.exists(name=name) else None
+        # )
+        if name not in self.ot_names:
+            await ctx.send(f":x: `{name}` not found!")
+            if names := self._find(name):
+                await self._send_paginated_embed(ctx, names, "Did you mean ❓")
```

Comment:
```suggestion
                await self._send_paginated_embed(ctx, names, "Did you mean one of the following?")
```
Maybe something like this? I don't love the ❓.